views/index.html

```diff
@@ -80,8 +80,9 @@
         <ul class="nav navbar-nav">
           <li class="active"><a href="/">Home</a></li>
           <li><a href="/dashboard">Dashboard</a></li>
-          <li><a href="https://github.com/FrancescoSTL/Site-Sonar">Download</a></li>
+          <li><a href="/privacy">Privacy</a></li>
```

Comment:
Add this here too? - [L#165](https://github.com/FrancescoSTL/Site-Sonar-Dashboard/blob/7e1ca358181cf3dad0c80f002a8ef28b82d09816/views/index.html#L165)